src/store/story.js

```diff
@@ -140,7 +140,7 @@ export default {
       }
     },
     valid: (state) => {
-      return state.frames.length >= 2 && state.frames.every( f => f.img && f.text )
+      return state.frames.length >= 2 && state.frames.every( f => f.img )
```

Comment:
Making it valid with text missing?

Reply:
Currently the Wikipedia article images are landing without caption, we should update that so all frames have text